apps/finance/app/src/lib/token-utils.js

```diff
@@ -1,2 +1,10 @@
+import { ETHER_TOKEN_VERIFIED_LIST } from './verified-tokens'
+
 export const ETHER_TOKEN_FAKE_ADDRESS =
   '0x0000000000000000000000000000000000000000'
+
+export const isTokenVerified = (tokenAddress, networkType) =>
+  networkType !== 'mainnet'
```

Comment:
Don’t we want the opposite?

Also we could do it like that:

```jsx
networkType !== 'mainnet' || ETHER_TOKEN_VERIFIED_LIST.has(tokenAddress.toLowerCase())
```

Reply:
Oops, forgot to revert that from testing :D. Good catch!